src/main/java/apoc/meta/Meta.java

```diff
@@ -430,6 +432,126 @@ private void collectStats(Collection<String> labelNames, Collection<String> relT
+     * supports flexible sampling options, and does not scan the entire database.  The result is producing a table of
+     * metadata that is useful for generating "Tables 4 Labels" schema designs for RDBMSs, but in a more performant way.
+     */
+    @Procedure
+    @Description("apoc.meta.nodeTypeProperties()")
+    public Stream<Tables4LabelsProfile.NodeTypePropertiesEntry> nodeTypeProperties(@Name(value = "config",defaultValue = "{}") Map<String,Object> config) {
+        MetaConfig metaConfig = new MetaConfig(config);
+        try {
+            return collectTables4LabelsProfile(metaConfig).asNodeStream();
+        } catch (Exception exc) {
+            exc.printStackTrace();
+        }
+
+        return null;
+    }
+
+    /**
+     * This procedure is intended to replicate what's in the core Neo4j product, but with the crucial difference that it
+     * supports flexible sampling options, and does not scan the entire database.  The result is producing a table of
+     * metadata that is useful for generating "Tables 4 Labels" schema designs for RDBMSs, but in a more performant way.
+     */
+    @Procedure
+    @Description("apoc.meta.relTypeProperties()")
+    public Stream<Tables4LabelsProfile.RelTypePropertiesEntry> relTypeProperties(@Name(value = "config",defaultValue = "{}") Map<String,Object> config) {
+        MetaConfig metaConfig = new MetaConfig(config);
+        try {
+            return collectTables4LabelsProfile(metaConfig).asRelStream();
+        } catch (Exception exc) {
+            exc.printStackTrace();
+        }
+
+        return null;
+    }
+
+    private Tables4LabelsProfile collectTables4LabelsProfile (MetaConfig config) {
+        Tables4LabelsProfile profile = new Tables4LabelsProfile();
+
+        Schema schema = tx.schema();
+
+        Map<String, Iterable<ConstraintDefinition>> relConstraints = new HashMap<>(20);
+
+        for (RelationshipType type : tx.getAllRelationshipTypesInUse()) {
+            List<ConstraintDefinition> tcd = new ArrayList<ConstraintDefinition>();
+            for (ConstraintDefinition cd : schema.getConstraints(type)) {
+                if (cd.isConstraintType(ConstraintType.RELATIONSHIP_PROPERTY_EXISTENCE)) {
+                    tcd.add(cd);
+                }
+            }
+            relConstraints.put(type.name(),tcd);
+        }
+
+        Map<String, Long> countStore = getLabelCountStore();
+
+        Set<String> includeLabels = config.getIncludesLabels();
+        Set<String> excludes = config.getExcludes();
+
+        Set<String> includeRels = config.getIncludesRels();
+        Set<String> excludeRels = config.getExcludeRels();
+
+        for (Label label : tx.getAllLabelsInUse()) {
+            String labelName = label.name();
+
+            if (excludes.contains(labelName)) {
+                // Skip if explicitly excluded
+                continue;
```

Comment:
`continue` is IMO bad practice. Instead use a `if (!badCondition) { /* do stuff */ }`

Reply:
I changed it and wrapped the code in an if predicate. I'm not too concerned with continue's usually, but I can see where it could be an issue. Performance wise this might save a few clock cycles too (not really sure)!